modules/rollup-plugin-curry/lib/plugin.js

```diff
@@ -19,9 +20,13 @@ export default function plugin () {
     generateBundle (_, files) {
       Object.entries(files).forEach(([ key, value ]) => {
         const { source } = value
-        if (key.endsWith('.d.ts') && source.indexOf('function') !== -1) {
+        if (key.endsWith('.d.ts') && source.includes(' function ')) {
+          const sourceRows = source.split('\n')
+          const newSource = sourceRows.reduce((acc, row) => (row.includes(' function ')
+            ? concat(acc, `${curryDefinition(row)}\n`)
```

Comment:
You can use string templates instead of concat: `${acc} ${curryDefinition(row)}\n`